test/support/page_objects/common.js

```diff
@@ -251,12 +251,16 @@ export default class Common {
   createErrorHandler(testObj) {
     const testName = (testObj.parent) ? [testObj.parent.name, testObj.name].join('_') : testObj.name;
     return error => {
-      const now = Date.now();
-      const fileName = `failure_${now}_${testName}`;
+      return this.remote.getCurrentUrl()
```

Comment:
[Chained operations](https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#chaining-operations) should be indented according to our style guide:

```
return this.remote.getCurrentUrl()
  .then(url => {
    PageObjects.common.debug(`Failure at URL ${url}`);
    const now = Date.now();
    const fileName = `failure_${now}_${testName}`;

    return this.saveScreenshot(fileName, true)
      .then(function () {
        throw error;
      });
  });
```